### ISSUE TITLE
installation python 2.7

### ISSUE THREAD

**Issue by amn3142** [gbrammer/grizli#33]
Running initial python setup.py install

```
 File "/home/nierenbe/repositories/anaconda2/envs/astroconda/lib/python2.7/site-packages/grizli-0.9.0_100_ga8aaa6b-py2.7-linux-x86_64.egg/grizli/fitting.py", line 172
    st.set_photometry(**phot, min_err=sys_err)
                            ^
SyntaxError: invalid syntax
```
Fixed by changing lines 170- 174 of grizli/fitting.py to:
```
        if phot is not None:
            if st is not None:
                st.set_photometry(min_err=sys_err,**phot)
        
            mb.set_photometry(min_err=sys_err, **phot)
            
```
And rerunning python setup.py install

**Comment by gbrammer** [comment]
Fixed in 416b66aae7fc527880d23c4a7defdae077ece07d.  Thanks, @amn3142!

I'd recommend installing with `environment_min.yml` as described in the installation instructions and `python>=3.5.5` since there's no telling how long the many STScI tools are going to continue to work in `2.7`.

**Comment by amn3142** [comment]
Yep!! I saw that 2.7 is ending. I'll just finish this project in 2.7 and then move on. *broken heart emoji*
